src/uct/ib/ud/base/ud_iface.c

```diff
@@ -526,6 +525,9 @@ UCS_CLASS_INIT_FUNC(uct_ud_iface_t, uct_ud_iface_ops_t *ops,
         self->tx.timer_backoff = config->timer_backoff;
     }
 
+    self->config.linger_timeout = ucs_time_from_sec(config->linger_timeout);
+    self->config.peer_timeout   = ucs_time_from_sec(config->peer_timeout);
```

Comment:
can we move it to line 499 along with rest of self->config initialization?

Reply:
done